lib/src/event_buffer.dart

```diff
@@ -97,7 +97,7 @@ class EventBuffer {
   Future<List<Event>> fetch(int count) async {
     assert(count >= 0);
 
-    final endRange = min(count, store!.length!);
+    final endRange = min(count, store!.length);
```

Comment:
Como não temos tratamento de erro caso `store` seja nula, acho que deveríamos utilizar `?` ao invés de `!`, por que se `store` for `null` e tentar acessar vai crashar e quem vai tratar o erro?